src/controllers/sandbox.ts

```diff
@@ -58,13 +59,15 @@ function cacheSandboxDescriptor(fetcher: (taskID: string) => Promise<SandboxDesc
 const sandboxCache = cacheSandboxDescriptor(async (taskID) => {
     const taskInfo = await getTaskInfo(taskID);
     const slaveState = await getMesosSlaveState(taskInfo.agent_url);
+    const status = (taskInfo.statuses.length > 0) ? taskInfo.statuses[taskInfo.statuses.length - 1] : 'UNKNOWN';
```

Comment:
I don't understand why don't we always use 'UNKNOWN' here

Reply:
because we want to know whether the task is still running or not.